src/autoscaler/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.1
 
 require (
 	code.cloudfoundry.org/cfhttp/v2 v2.1.0
-	code.cloudfoundry.org/clock v1.1.0
+	code.cloudfoundry.org/clock v1.2.0
```

Comment:
Any idea why this wasn't picked up by Renovate? 🤔 💭

Reply:
Due to limited availability of GitHub Actions runners, we have limited the maximum number of concurrent Renovate PRs to `2`. The `clock` update is in the list at: https://github.com/cloudfoundry/app-autoscaler-release/issues/1100

Please feel free to trigger missing updates from the issue if their update is pressing.

Reply:
Thanks for that, I totally forgot about it! ❤️